src/resolv/stub/mod.rs

```diff
@@ -141,6 +141,7 @@ impl StubResolver {
         search_host(&self, qname).await
     }
 
+    /// See the documentation for the [`lookup_srv`] function for details.
```

Comment:
Could you add a short one-line summary before this note? Something trivial like ‘Performs an SRV lookup using the stub resolver’ will do.